website/client/src/components/static/communityGuidelines.vue

```diff
@@ -7,7 +7,10 @@
       {{ $t('commGuideHeadingWelcome') }}
     </h2>
     <div class="media align-items-center">
-      <img src="~@/assets/images/community-guidelines/intro.png">
+      <img
+        src="~@/assets/images/community-guidelines/intro.png"
+        :alt="$t('commGuideHeadingWelcome')"
```

Comment:
Hmm I'm not 100% sure this file needs alt tag, the info added to the alt tags is already part of the text and that can be read with screen readers, I'm not sure duplicating it is a good idea . Opinions @Alys @Tressley ?